superset/viz.py

```diff
@@ -1208,14 +1208,21 @@ class DistributionPieViz(NVD3Viz):
     verbose_name = _("Distribution - NVD3 - Pie Chart")
     is_timeseries = False
     fieldsets = ({
-        'label': None,
+        'label': 'Criterion',
```

Comment:
Do we really need the label here? Criterion does not look like the right one anyway.